src/android/build-extras.gradle

```diff
@@ -19,7 +19,7 @@ static def mostRecentVersion(List<String> versions) {
 }
 
 static def getHyperSDKVersion(rootProject) {
-    def hyperSdkVersion = '2.1.13'
+    def hyperSdkVersion = '2.1.20-rc.06'
```

Comment:
Update SDK version here to 2.1.20-rc.10, or preferably, 2.1.20 once it is released by today/tmrw